cmd/pwmTools/shuffle.go

```diff
@@ -0,0 +1,86 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/vertgenlab/gonomics/exception"
+	"github.com/vertgenlab/gonomics/fileio"
+	"github.com/vertgenlab/gonomics/motif"
+	"log"
+	"math/rand"
+	"os"
+)
+
+type ShuffleSettings struct {
+	InFile     string
+	OutFile    string
+	NumShuffle int
+	SetSeed    int64
+}
+
+func shuffleUsage(shuffleFlags *flag.FlagSet) {
+	fmt.Printf("pwmTools shuffle - a tool for producing shuffled control motifs from input motifs.\n" +
+		"pwmTools shuffle in.pwm out.pwm\n" +
+		"options:\n")
+	shuffleFlags.PrintDefaults()
+}
+
+func parseShuffleArgs() {
+	var expectedNumArgs int = 2
+	var err error
+	shuffleFlags := flag.NewFlagSet("shuffle", flag.ExitOnError)
+	var numShuffle *int = shuffleFlags.Int("numShuffle", 10, "Specify the number of desired shuffle motifs per input motif.")
```

Comment:
Did you mean to write "Specify the number of desired **shuffled** motifs..." instead?